discovery-provider/nginx_conf/nginx.conf

```
@@ -84,9 +101,14 @@ http {
         location ~* .*/search/.* {
             access_by_lua_block {
                 local main = require "main"
+                ngx.shared.request_count:incr("request-count", 1, 0)
```

```suggestion
                local rcount_key = "request-count"
                local rcount_step_value = 1
                local rcount_init_value = 0
                ngx.shared.request_count:incr(rcount_key, rcount_step_value, rcount_init_value)
```

can we do something like this? i had to look up the [method signature](https://github.com/rainingmaster/lua-shdict-nginx-module#incr) to confirm what the `0` was doing and would be good to have named vars so subsequent readers don't have to do the same ;)